app/lib/aid_application_cleaner.rb

```diff
@@ -0,0 +1,14 @@
+class AidApplicationCleaner
+  def clean_production
+    if Rails.env.production?
+      apps_to_delete = AidApplication.where(submitted_at: nil).where('created_at < ?', 1.day.ago)
+      apps_to_delete.destroy_all
+    end
+  end
+
+  def clean_demo
+    if Rails.env.demo?
+      AidApplication.destroy_all
+    end
+  end
```

Comment:
For cleaning Demo, I'd like to do something more comprehensive like on GCF with this Rake task (I wouldn't object if you copy pasted it; it would need to be set up with the Heroku Scheduler add-on rather than inside our codebase in Rufus-Scheduler): https://github.com/codeforamerica/gcf-backend/blob/master/lib/tasks/heroku.rake#L17-L30